SharePointFramework/ProjectWebParts/src/components/ProjectStatus/index.tsx

```diff
@@ -69,11 +69,14 @@ export class ProjectStatus extends React.Component<IProjectStatusProps, IProject
           (report) => report.id === parseInt(selectedReportUrlParam, 10)
         )
       }
+      const newestReportId = data.reports.length > 0 ? Math.max.apply(Math, data.reports.map(report => { return report.id; })) : 0;
```

Comment:
The reports are sorted (I think?), so the newest report will always be on the same index in the array I'd reckon?

Reply:
> The reports are sorted (I think?), so the newest report will always be on the same index in the array I'd reckon?

Yeah, they appear to be sorted. Changed to select the first element in array